content/etc/init/openbsd-inetd-reconf-on-netchange.conf

```diff
@@ -0,0 +1,5 @@
+start on net-device-up
```

Comment:
Wouldn't it not better to start if service openbsd-inetd is already running?

start on net-device-up and started openbsd-inetd